### PR TITLE
Add move last cursor left and right

### PR DESCRIPTION
**Pull request by lilole**
Howdy. I finally got around to adding this. It was a feature I've been missing for a long time from Atom.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,6 +61,20 @@
...
+				"mac": "cmd+alt+left",
+				"linux": "shift+alt+left",
+				"when": "editorTextFocus"
+			},
+			{
+				"command": "smartmulticursor.lastCursorRight",
+				"key": "ctrl+alt+right",
+				"mac": "cmd+alt+right",
+				"linux": "shift+alt+right",
+				"when": "editorTextFocus"
```
Comment (idanpa) [inline package.json]: I don't think it is good to override "Move Editor into Next/Previous Group". Better remove it and let the user rewrite the keybinding if desired.

File: package.json [hunk 2]
```diff
@@ -10,8 +10,8 @@
 		"url": "https://github.com/idanpa/vscode-smartmulticursor"
 	},
 	"bugs": {
-        "url": "https://github.com/idanpa/vscode-smartmulticursor/issues"
-    },
+		"url": "https://github.com/idanpa/vscode-smartmulticursor/issues"
```
Comment (idanpa) [inline package.json]: remove

**Review by idanpa — CHANGES_REQUESTED** [review]
The idea of adding this functionality seems ok to me. Just don't like the overriding of existing keybinding

File: package.json (hunk 1 above)
Comment (lilole) [reply package.json]: No prob at all. Will remove.

File: package.json (hunk 2 above)
Comment (lilole) [reply package.json]: Will remove. I think my editor wanted to align things.

File: package.json (hunk 1 above)
Comment (lilole) [reply package.json]: Fixed in https://github.com/idanpa/vscode-smartmulticursor/pull/8/commits/51c4b75d76de036382f2ebd8037b00877006e529

File: package.json (hunk 2 above)
Comment (lilole) [reply package.json]: Fixed in https://github.com/idanpa/vscode-smartmulticursor/pull/8/commits/51c4b75d76de036382f2ebd8037b00877006e529

**Comment by lilole** [comment]
@idanpa If it helps I can take a whack at updating the README. Just let me know.

**Comment by idanpa** [comment]
@lilole Thanks a lot for your contribution! ❤️

**Pull request merged**